src/main/scala/pipelines/text/AmazonReviewsPipeline.scala

```diff
@@ -71,11 +72,11 @@ object AmazonReviewsPipeline extends Logging {
     val conf = new SparkConf().setAppName(appName)
     conf.setIfMissing("spark.master", "local[2]") // This is a fallback if things aren't set via spark submit.
 
-    val sc = new SparkContext(conf)
+    val spark = SparkSession.builder.config(conf).getOrCreate()
```

Comment:
That said, I agree that the examples should be consistent. In that case all the data loaders should also take a `SparkSession`. I have created #291 to track.